JSIL.mscorlib/JSIL/System/NumberFormatter.Data.cs

```diff
@@ -0,0 +1,916 @@
+﻿//
+// mono/mono/metadata/number-formatter.h
```

Comment:
Hmm, good catch. Looks like this file, as it located in mono/mono folder is licensed only with LGPL or commercial license.
It is really bad, so we should check if it was previously located under anther directory pr ask them for using this file with another license :(